Src/coffeescript/cql-execution/test/datatypes/uncertainty.coffee

```diff
@@ -24,6 +24,10 @@ describe 'Uncertainty', ->
     should(everything.low).be.null
     should(everything.high).be.null
 
+    differentTypes = new Uncertainty(1, "1")
+    differentTypes.low.should.eql 1
+    differentTypes.high.should.eql "1"
```

Comment:
This is fine in that it documents the current behavior, but _really_, I don't think it should be allowed to have the low and high as different types.  Is there a use case?  That should probably _throw_...

If you are making changes anyway, can you just comment that this probably _shouldn't_ be supported?